README.md

```diff
@@ -32,7 +32,7 @@ EventMachine has been around since the early 2000s and is a mature and battle-te
 
 ## What platforms are supported by EventMachine? ##
 
-EventMachine supports Ruby >= 1.8.7 and <= 2.2 REE, JRuby and **works well on Windows** as well
+EventMachine supports Ruby >= 1.8.7 and <= 2.3 REE, JRuby and **works well on Windows** as well
```

Comment:
Let's reword it to be less strict about the end point:
`EventMachine supports Ruby 1.8.7 through 2.3, REE, JRuby and...`